src/style.css

```diff
@@ -1 +1,42 @@
-/* add styles here */
+.page {
+  margin: 0;
```

Comment:
Avoid using the `margin` property to zero out the margin on the `body` element. Instead, set the `margin` property on the `body` tag directly in your CSS to ensure consistency and to avoid potential issues with global styles.